mrc-ide/hint
# mrc-3109 Display warnings from input validation

This PR displays warnings from SAP and GenericChart in Review input step, I merged the warnings from different data points such as SurveryResponse, ProgramResponse, AncResponse and TimeSeriesResponse. 

Even thou we currently don't expect data from GenericCharts, it will be populated once we do. 

Warnings will be displayed upon reaching review input step and once timeSeries is triggered, the additional chart warnings will be appended.

To test
* Start App
* Load data and navigate to review input
* Warnings should be displayed

If you want to test genericChart warnings are also appended, you can manually add test warnings here https://github.com/mrc-ide/hint/blob/a5ee5cf0dbe91b74ebbcfb9b86b9c8d99bed14ad/src/app/static/src/app/store/genericChart/actions.ts#L46

`const testWarnings = [{"text": "test genericChart", "locations": ['review_inputs']}]`

replace `data.warnings` with `testWarnings`

NB: DO NOT MERGE
depends on: https://github.com/mrc-ide/hintr/pull/370 to be merged
once hintr equivalent is merged then change hintr branch to master 

## Type of version change
_Delete as appropriate_

Minor 

## Checklist
